guidelines/enhancement_template.md

```diff
@@ -71,6 +71,7 @@ around the enhancement process.
 - [ ] Enhancement is `implementable`
 - [ ] Design details are appropriately documented from clear requirements
 - [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
```

Comment:
This is vague to me.  I like the idea, but I think there are probably 200 different ideas of what this means.

Specifically, I'm not sure if this is 'As an managed OpenShift SRE' or 'As a customer cluster operator'.  I know it's partly both, but focusing on one or the other might help engineers understand what they actually need to be thinking about.

Reply:
Need to decide how much goes here vs another doc.  I can link out, if you think that's the right approach.

Reply:
Summarizing our offline discussion; the enhancement template doesn't include details or links to other implementation guides, so we will continue that pattern here.  I intend to find teams (maybe review some existing enhancements) to work with directly to gather some more detail.

Reply:
I think we should change that pattern, but ok for now given there's an effort to define these somewhere.